### PR TITLE
Fix total_regular_file_size doesn't exist & cleanup ota_client.py

### PR DESCRIPTION
**Pull request by Bodong-Yang**
- fix [JIRA] (T4PB-19045) [OTA] [refactor3] total_regular_file_size doesn't exist
- update the way of converting internal status class to compiled v2 proto defined structure
- ota_client.OTAUpdater: move boot_control pre_update execution to _pre_update method

### CODE REVIEW

File: app/update_stats.py [hunk 1]
```diff
@@ -146,16 +161,20 @@ def collector(self):
...
                     for st in self._staging:
-                        _suffix = st.op.value
-                        if _suffix in {"copy", "link", "download"}:
+                        if (
+                            isinstance(st.op, RegProcessOperation)
+                            and st.op != RegProcessOperation.OP_UNSPECIFIC
+                        ):
+                            _suffix = st.op.value
                             staging_storage.regular_files_processed += 1
+                            staging_storage.total_regular_file_size += st.size
```
Comment (obi-t4) [inline app/update_stats.py:170]: total_regular_file_size is static value. not dynamic value.

Comment (Bodong-Yang) [reply app/update_stats.py:170]: Oh, I miss understanding this field.

Comment (Bodong-Yang) [reply app/update_stats.py]: fixed in [d72a041](https://github.com/tier4/ota-client/pull/137/commits/d72a041b118aa22bcf01dcf6f50a80a321de2365), the real cause of the problem is that I set the total_regular_file_size before launching the update_stats_collecter.

**Review by obi-t4 — APPROVED** [review]
LGTM :+1:

**Pull request merged**